### PR TITLE
i18n(es):  Update `docker.mdx`, `view-transitions.mdx` & `cloudflare.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Update `docker.mdx`, `view-transitions.mdx` & `cloudflare.mdx` based on these 3 commits 1ceebd6179e64e43e32ddd5b053167ddd0b46e99, 5f6e71a41f2157d0df701091fa5fe46aa9a8b436 & dd2dc55c9b97bb40b10338bde7b6edbedc7a466c

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 1]
```diff
@@ -86,12 +86,59 @@ Después es posible actualizar el script de vista previa en tu `package.json` a
...
 ```
 
+Desde un endpoint:
+
+```js
+// src/pages/api/someFile.js
+export function get(context) {
+  const runtime = context.locals.runtime;
+
+  return new Response('Some body');
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:102]:
```suggestion
  return new Response('Algún cuerpo');
```

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 2]
```diff
@@ -86,12 +86,59 @@ Después es posible actualizar el script de vista previa en tu `package.json` a
...
+}
+```
+
 Dependiendo del modo de adaptador que estés utilizando (avanzado = worker, directorio = páginas), el objeto runtime tendrá una apariencia un poco diferente debido a las diferencias en la API de Cloudflare.
 
+Si estás usando el tiempo de ejecución `advanced`, puedes escribir el objeto `runtime` de la siguiente manera:
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:108]:
```suggestion
Si estás usando el runtime`advanced`, puedes escribir el objeto `runtime` de la siguiente manera:
```

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 3]
```diff
@@ -86,12 +86,59 @@ Después es posible actualizar el script de vista previa en tu `package.json` a
...
+  interface Locals extends AdvancedRuntime {
+    user: {
+      name: string;
+      surname: string;
+    };
+  }
+}
+```
+
+Si estás usando el tiempo de ejecución `advanced`, puedes escribir el objeto `runtime` de la siguiente manera:
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:125]:
```suggestion
Si estás usando el runtime `directory`, puedes escribir el objeto `runtime` de la siguiente manera:
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**